CHANGELOG.md

```diff
@@ -1,5 +1,225 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.11
+
+###### _Sep 26, 2020_
+
+Big thanks to the 29 contributors who made this release possible.
+Here are some highlights ✨:
+
+- 👩‍🎨 A first iteration on the new styling solution.
+
+  You can find a [new version](https://next.material-ui.com/components/slider-styled/) of the slider in the lab powered by [emotion](https://emotion.sh/).
+
+  In the event you are already using styled-components in your application, you can swap emotion for styled-components 💅. Check [this CodeSandbox](https://codesandbox.io/s/sliderstyled-with-styled-components-2hjs0?file=/src/demo.tsx) for a demo, it relies on aliases to prevent any bundle size overhead.
```

Comment:
```suggestion
  In the event that you are already using styled-components in your application, you can swap emotion for styled-components 💅. Check [this CodeSandbox](https://codesandbox.io/s/sliderstyled-with-styled-components-2hjs0?file=/src/demo.tsx) for a demo. It relies on aliases to prevent any bundle size overhead.
```